File: lib/ts/src/UApiSchemaFiles.ts
import { FsModule, PathModule } from './fileSystem';
import { getSchemaFileMap } from './internal/schema/GetSchemaFileMap';

export class UApiSchemaFiles {
    filenamesToJson: Record<string, string>;

    constructor(directory: string, fs: FsModule, path: PathModule) {
        this.filenamesToJson = getSchemaFileMap(directory, fs, path);
    }
}

File: lib/java/src/main/java/uapi/UApiSchemaFiles.java
package uapi;

import static uapi.internal.schema.GetSchemaFileMap.getSchemaFileMap;

import java.util.Map;

public class UApiSchemaFiles {

    public final Map<String, String> filenamesToJson;

    public UApiSchemaFiles(String directory) {
        this.filenamesToJson = getSchemaFileMap(directory);
    }
}
